src/ui/components/customfields/CustomFields.js

```diff
@@ -28,15 +29,15 @@ function CustomFields(props) {
         ? <SimpleEmptyState message="No Custom Fields to show." />
         : (
             <div style={{width: "100%", marginTop: 0}}>
-                {customFields.map((customField, index) => (
+                {customFields.map((customField, index) => (!customField.code.startsWith('MTFX') || !isCernMode) &&
```

Comment:
I would just add a comment explaining why we are doing this.

Also, what about using `.filter(..)` instead?